src/inmanta/logging.py

```diff
@@ -788,6 +791,27 @@ def apply_options(self, options: Options, component: str | None = None, context:
         Used to select which config file option to use (logging.component)
         :param context: context variables to use if the config file is a template
         """
+
+        def user_defined_options() -> Optional[str]:
+            """
+            Returns a string with the options (excluding "--verbose" and "--logging-config") that were set by the user
+            """
+            ignore_keys = ["verbose", "logging_config"]
+
+            option_to_cli = {
+                "log_file": "--log-file",
+                "log_file_level": "--log-file-level",
+                "timed": "--timed-logs",
+                "keep_logger_names": "--keep-logger-names",
+            }
+            ignored_options = [
+                f"{option_to_cli[key]} {value}"
+                for key, value in options.__dict__.items()
+                if key not in ignore_keys and key in options
```

Comment:
```suggestion
                for key, value in options.__dict__.items()
                if key not in ignore_keys
```
Isn't this redundant?

Reply:
Ah no, `options.__dict__` are all possible options while `options` are the defined ones?

Reply:
Yes, I'm reworking the logic a bit to be (in my opinion) simpler